app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/Place.java

```diff
@@ -65,19 +65,19 @@ public void setId (String id)
         this.id = id;
     }
 
-    public String getPlace_id ()
+    public String getPlaceId()
     {
-        return place_id;
+        return placeId;
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.